app.js

```diff
@@ -31,4 +31,20 @@ client.on('messageCreate', messageHandler);
 
 client.on('messageDelete', logDeletedMessages);
 
+process.on('unhandledRejection', async (error) => {
```

Comment:
In order to keep things nice and tidy, I think it may be best to follow the pattern in the other handlers and move this to it's own file in the `./handlers` folder then import and reference it here.

Reply:
Good idea. I restructured it like you suggested in the latest commit.